src/main/java/com/um/inventoryservice/util/EntityDTOUtil.java

```diff
@@ -89,7 +89,12 @@ public static String generateEmployeeString() {
         return (String.format("%05d", number));
     }
 
-    public static String generateUUID(){
-        return UUID.randomUUID().toString();
+    public static String generateClientString() {
```

Comment:
After further development, this will be reprogrammed for admin to just enter a client Id, correct?

Reply:
Yes, there will be no time to complete the client side so we switched it to be for admin.